frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistSceneLogic.ts

```diff
@@ -126,11 +127,21 @@ export const sessionRecordingsPlaylistSceneLogic = kea<sessionRecordingsPlaylist
     })),
 
     listeners(({ actions, values }) => ({
-        getPlaylistSuccess: () => {
+        getPlaylistSuccess: ({ playlist }) => {
             if (values.playlist?.derived_name !== values.derivedName) {
                 // This keeps the derived name up to date if the playlist changes
                 actions.updatePlaylist({ derived_name: values.derivedName }, true)
             }
+
+            if (playlist.filters) {
```

Comment:
since we're going to split playlists (have filters) and collections (have pinned recordings) i'll not count the second group here